src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+enum Color {
+    BLUE,
+    RED,
+    GREEN
+}
```

Comment:
The enum `Color` should have a private constructor to prevent instantiation. Also, enum types should override the `toString()` method only if they are providing custom String representation. Otherwise, it's better to use the `name()` method to get the enum constant's name.